desci-contracts/scripts/seed.sh

```diff
@@ -28,7 +28,7 @@ check() {
   done
 
   echo "[seed:$CONTRACT_NAME] deployment found, killing ganache..."
-  ps aux | grep ganache-cli | grep -v grep | awk '{print $2}' | xargs -r -t kill || echo "[seed:$CONTRACT_NAME] ganache not running"
+  pkill -f "npm exec ganache" || echo "[seed:$CONTRACT_NAME] ganache not running"
```

Comment:
This was broken (for me, at least) because the binary is called just `ganache` now. Also, due to the `-r` flag to xargs which makes it always succeed even if there are no arguments to apply, it never logged the failure.